cmd/otelcol/config/collector/gateway_config.yaml

```diff
@@ -57,6 +57,8 @@ receivers:
     # Whether to preserve incoming access token and use instead of exporter token
     # default = false
     #access_token_passthrough: true
+  splunk_hec/logs:
+    endpoint: 0.0.0.0:8088
```

Comment:
Do we need to expose it if we recommend using otlp between the agent and gateway?

Reply:
We might need to expose it for profiling, which uses HEC.

Reply:
> We might need to expose it for profiling, which uses HEC.

Really? Do profiling libraries send HEC format? I was under impression it's OTLP